src/renderers/art/ReactARTFiberEntry.js

```diff
@@ -509,9 +509,7 @@ const ARTRenderer = ReactFiberReconciler({
     return emptyObject;
   },
 
-  scheduleAnimationCallback: ReactDOMFrameScheduling.rAF,
-
-  scheduleDeferredCallback: ReactDOMFrameScheduling.rIC,
+  scheduleCallback: ReactDOMFrameScheduling.rIC,
```

Comment:
`scheduleAsyncCallback`? Seems redundant, but so does "deferred" :D Will we ever schedule more than one type of callback?